.github/workflows/maven-deploy.yml

```diff
@@ -0,0 +1,50 @@
+name: Deploy with Maven
+
+on:
+  workflow_dispatch:
+    branches:
+      - 'develop'
+      - 'master'
+      - 'release/*'
+      - 'test/ci'
+    inputs:
+      style:
```

Comment:
I am not sure `style` is the right word for this. I would prefer `target` or `release_type` or something similar.

Reply:
so release_type - I suggest to avoid possible key words.